core/src/main/java/tc/oc/pgm/shops/ShopKeeper.java

```diff
@@ -0,0 +1,65 @@
+package tc.oc.pgm.shops;
+
+import static tc.oc.pgm.util.bukkit.BukkitUtils.colorize;
+
+import com.google.common.collect.ImmutableList;
+import javax.annotation.Nullable;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.bukkit.metadata.FixedMetadataValue;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.points.PointProvider;
+import tc.oc.pgm.util.nms.NMSHacks;
+
+public class ShopKeeper {
+
+  public static final String METADATA_KEY = "SHOP_KEEPER";
+
+  private final String name;
+  private final ImmutableList<PointProvider> location;
```

Comment:
`
```suggestion
  private final Collection<PointProvider> location;
```

Reply:
I would try to simply change this to directly PointProvider, not a collection. Multiple shouldn't be supported anyways